Applications/web_portal/web_app/library/source/pages/dashboard_page/dashboard_page.tsx

```diff
@@ -1,16 +1,88 @@
+import {css, StyleSheet} from 'aphrodite';
 import * as React from 'react';
+import {BurgerButton, HBoxLayout, Padding, VBoxLayout} from '../..';
+import {NotificationButton} from './notification_button';
 
 export interface Properties {}
 
-export interface State {}
+export interface State {
+}
 
 /** Displays the main dashboard. */
 export class DashboardPage extends React.Component<Properties, State> {
-  constructor(properties: Properties) {
-    super(properties);
+  constructor(props: Properties) {
+    super(props);
   }
 
   public render(): JSX.Element {
-    return <div>Dashboard</div>;
+    return (
+      <VBoxLayout width='100%' height='100%'>
+        <div className={css(DashboardPage.STYLE.header)}>
+          <HBoxLayout className={css(DashboardPage.STYLE.leftHeader)}
+              width='136px' height='60px'>
```

Comment:
Remove the use of this leftHeader STYLE and instead layout the items strictly using Paddings and BoxLayout elements.